src/api/cli/integration/integration_test.go

```diff
@@ -0,0 +1,312 @@
+package cli_integration_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/skycoin/skycoin/src/api/webrpc"
+	"github.com/skycoin/skycoin/src/wallet"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	binaryName = "skycoin-cli"
+	walletName = "integration_test.wlt"
+
+	testModeStable = "stable"
+	testModeLive   = "live"
+)
+
+var (
+	binaryPath string
+	walletDir  string
+)
+
+var update = flag.Bool("update", false, "update golden files")
+
+func TestGenerateAddresses(t *testing.T) {
+	if !doLive(t) && !doStable(t) {
```

Comment:
Need to change this to `doLiveOrStable` like https://github.com/skycoin/skycoin/pull/989